### DISCUSSION TITLE
Issue raised has been closed and merged! What next?!

### DISCUSSION THREAD

**Discussion by Braedencraig** [vercel/next.js#19019] [Help] [answered]
RE: This issue, https://github.com/vercel/next.js/issues/18563, it is now closed. Yahooo! Thanks y'all! I see the milestone for it is Iteration 12, which is due November 16th. Does this mean, that after November 16th, i'll be able to update my version of Next and it will include that fix I am expecting? Or is the Iteration date, simply just a projected sprint end date, and not the actual release date for the fixes on that Iteration?

**Comment by Timer** [comment, marked as answer]
You can upgrade to `next@canary` and start using it immediately!
